TACC/Core-CMS-Custom
# chore(lccf): cleanup

## Overview / Changes

- Add commented CSS that might be used.
- Remove unused snippets.

## Related

LCCF will launch with a Portal. I will replace CSS snippets on three staging sites and current prod site with a link to this updated stylesheet.

## Testing

1. Deploy LCCF with CMS Settings

    ```py
    PORTAL_STYLES = [{
        "is_remote": True,
        "path": "https://cdn.jsdelivr.net/gh/TACC/Core-CMS-Custom@chore/lccf-cleanup/lccf_assets/header.css",
    }]
    ```

    and Portal settings

    ```css
    _PORTAL_CSS_FILENAMES = [
        "https://cdn.jsdelivr.net/gh/TACC/Core-CMS-Custom@chore/lccf-cleanup/lccf_assets/header.css"
    ]
    ```

2. Verify header is tall (if stylesheet loads[^1]).

## UI

| `lccf` CMS | `lccf-portal` (CMS) |
| - | - |
| <img width="845" alt="lccf" src="https://github.com/user-attachments/assets/ca3b8180-2d94-42fe-bbd0-d0fda47dc257"> | <img width="845" alt="lccf-portal" src="https://github.com/user-attachments/assets/d06c1fc6-fce4-4f5b-90ed-f3b96b68eaf2"> |

## Notes

[^1]: Stylesheet is not loading on portal. That is not a bug with this PR, which is just to supply a CDN.
